build.sbt

```diff
@@ -1,29 +1,40 @@
+lazy val scalaTestVersion = "3.2.3"
+
+lazy val commonSettings = Seq(
+  crossScalaVersions += "3.0.0-M1",
```

Comment:
shouldn't this also include 2.13.3?

Reply:
not sure. It's `+=` not `:=`. `show crossScalaVersions` shows both. But now I'm confused; I thought I did this to include the ones of the modules plugin, but I don't see 2.11, 2.12. Is it possible that this overrides the defaults? Also, should 3.0.0-M1 be in the modules plugin?

Reply:
Yes, it seems to override despite `+=`. So I will add _all_ supported versions: 2.11.12, 2.12.12, 2.13.3, 3.0.0-M1.

Reply:
if you want to drop 2.11 and 2.12, feel free to do so.

Reply:
oh, also, I forgot: the current sbt-scala-module includes sbt-travisci, so that means that `crossScalaVersions` gets set by reading `.travis.yml`, so you shouldn't be setting either `scalaVersion` or `crossScalaVersions` in `build.sbt` at all!

Reply:
> should 3.0.0-M1 be in the modules plugin?

not sure what you mean

Reply:
#127 addresses this